New attributes for eduTEAMS login

licehammer opened the pull request.

zlamalp commented on perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_def_login_namespace_eduteams_persistent_shadow.java:
```diff
@@ -0,0 +1,96 @@
+
+	private final static Logger log = LoggerFactory.getLogger(urn_perun_user_attribute_def_def_login_namespace_eduteams_persistent_shadow.class);
+	private final static String attrNameEduteams = "login-namespace:eduteams-persistent-shadow";
+	private final static String eduteamsAttributesPropertiesFile = "eduteams-attributes.properties";
```
I don't see any issue with it either. Maybe you could introduce entityless attribute(s) for it, but it`s probably easier to use it the way you have it.

licehammer commented:
I fixed the problems. I decided to drop the configuration file. For now, it's not needed. Also, I've added additional similar attributes to this pull-request.

zlamalp commented:
Seems OK to me now. I will make it part of 3.3.1.
